packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -662,6 +671,128 @@ export const startLienBridge = async ({
   lienBridge.resolve(reporter);
 };
 
+/**
+ * @param {EconomyBootstrapPowers} powers
+ * @param {object} config
+ * @param {any} [config.auctionParams]
+ */
+export const startAuction = async (
```

Comment:
similar to the `auctioneerKit` request,
```suggestion
export const startAuctioneer = async (
```

the _auction_ isn't being started by this and there will be many.

Reply:
done